Gruntfile.js

```diff
@@ -57,18 +57,36 @@ module.exports = function(grunt) {
             production: {
                 options: {
                     bucket: 'gdn-cdn',
-                    params: {
-                        CacheControl: 'max-age=604800'
-                    }
                 },
                 files: [{
                     expand: true,
                     cwd: dir,
-                    src: ['**/*.json', '**/source/*'],
-                    dest: remoteDir
+                    src: ['**/*.json'],
+                    dest: remoteDir,
+                    params: {
+                        CacheControl: 'max-age=60'
+                    }
+                },{
+                    expand: true,
+                    cwd: dir,
+                    src: ['**/source/*'],
+                    dest: remoteDir,
+                    params: {
+                        CacheControl: 'max-age=604800'
+                    }
                 }]
             }
         },
+        hash: {
```

Comment:
I hear that [grunt-asset-hash](https://www.npmjs.com/package/grunt-asset-hash) is the most awesome hash plugin in the grunty-sphere...